### DISCUSSION TITLE
[OP Stack Deployment] Would ideally fund an account on L1 Sepolia to run a testnet

### DISCUSSION THREAD

**Discussion by enrlabs** [ethereum-optimism/developers#21] [Deploying the OP Stack] [answered]
### Issue Description

Good evening, 

if we want to build a blockchain testnet based on OPStack, how much ETH is needed to run a public testnet for 5 months? assuming that the documentation at least recommends 17 ETH to fund 4 addresses. then 17 ETH will be used up for how long is the estimate? and ideally how much ETH will we hold during the testnet for 5 months?

**Comment by opfocus** [comment, marked as answer]
I haven't done the math, but 17 ETH should run for a long time. I'm not sure if it can run for 5 months. Two things to keep in mind are that it might be best to avoid using Goerli and opt for Sepolia for the L1datalayer. Another thing to explore is the local test environment, [Devnet](https://github.com/ethereum-optimism/developers/discussions/17#discussioncomment-7399981)

**Reply by enrlabs** [reply]
I quite understand this situation. maybe I need to try it. does optimism have bulk faucet? because sepolia faucet so long we get and don't have gitcoin passport for mining. if you have form to get sepolia bulk faucet please let me know sir. thanks.

**Reply by opfocus** [reply]
It's not open yet, but there might be a Sepolia faucet coming online soon. You can leave your address

**Reply by enrlabs** [reply]
Address : 0x399F60dF394A8b982Fba42990d48Ef3bB25435De

im waiting to get faucet sepolia :+1:

**Reply by opfocus** [reply]
https://sepolia.etherscan.io/tx/0x61fd3e4fc649f1f0029dfe757d06eada86cff0ef58fa0ba3a67748e033ba11ff

**Reply by enrlabs** [reply]
thankyou sir
